server/game/cards/05-DT/Card322.js

```diff
@@ -0,0 +1,27 @@
+const Card = require('../../Card.js');
+
+class Card322 extends Card {
+    //Play: For the remainder of the turn, after you play another card, exhaust a creature.
+    setupCardAbilities(ability) {
+        this.play({
+            gameAction: ability.actions.forRemainderOfTurn((context) => ({
+                when: {
+                    onCardPlayed: (event) =>
+                        event.player === context.player && event.card !== context.source
+                },
+                preferActionPromptMessage: true,
+                gameAction: ability.actions.exhaust({
+                    promptForSelect: {
```

Comment:
I think you can use target here instead of promptForSelect (pfs)

Reply:
Yeah, I think I was trying to do that at first but it didn't work at the time, and there's a similar workaround in place for Commandeer. I ran into this issue when working on the parser, but I think using target here means the engine interprets it as the target of the forRemainderOfTurn action, then puts it in an array etc. so it's tricky to update lasting effect action to support it

Reply:
Tricky, indeed. I see what you are saying.